WordPress/src/main/java/org/wordpress/android/ui/history/HistoryDetailContainerFragment.java

```diff
@@ -95,47 +194,38 @@ public boolean onOptionsItemSelected(MenuItem item) {
 
             getActivity().setResult(Activity.RESULT_OK, intent);
             getActivity().finish();
+        } else if (item.getItemId() == R.id.history_toggle_view) {
+            mVisualTitle.setText(isInVisualPreview() ? null : mRevision.getPostTitle());
+            mVisualContent.fromHtml(
+                    isInVisualPreview() ? "" : StringUtils.notNullStr(mRevision.getPostContent()), false);
```

Comment:
The crossfade from HTML view to visual view looks great!  The crossfade from visual view to HTML view has a minor oddity.  Since we reset the `mVisualTitle` and `mVisualContent` text with `null` and `""` when toggling from visual view to HTML view, those fields become blank showing just a white background at the beginning of the animation creating a blink-like transition.  What about changing the `else if` branch in the `onOptionsItemSelected` method to something like this?

```
        } else if (item.getItemId() == R.id.history_toggle_view) {
            if (isInVisualPreview()) {
                mPreviousButton.setEnabled(mPosition != 0);
                mNextButton.setEnabled(mPosition != mAdapter.getCount() - 1);
            } else {
                mPreviousButton.setEnabled(false);
                mNextButton.setEnabled(false);
                mVisualTitle.setText(mRevision.getPostTitle());
                mVisualContent.fromHtml(StringUtils.notNullStr(mRevision.getPostContent()), false);
            }

            crossfadePreviewViews();
        }
```

That will fix the slightly jarring transition from visual view to HTML view as well the `mPreviousButton` and `mNextButton` state bug.  See the animation below for illustration.

Before|After
-|-
<img src="https://user-images.githubusercontent.com/3827611/48498536-154ddf80-e7fc-11e8-8ef9-3b35a60a60b0.gif" width="320">|<img src="https://user-images.githubusercontent.com/3827611/48498543-1b43c080-e7fc-11e8-85fb-22009c46ebea.gif" width="320">